packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -1779,27 +1825,47 @@ space
       it('logs out $el when existing $el is found even on failure', function (done) {
         const button = cy.$$('#button')
 
-        cy.on('fail', (err) => {
-          expect(this.lastLog.get('state')).to.eq('failed')
-          expect(this.lastLog.get('error')).to.eq(err)
-          expect(this.lastLog.get('$el').get(0)).to.eq(button.get(0))
-          const consoleProps = this.lastLog.invoke('consoleProps')
+        cy.once('fail', (err) => {
+          assertLogLength(this.logs, 2)
+
+          const containsLog = this.logs[0]
+          const assertionLog = this.logs[1]
+
+          expect(err.message).to.contain(`Expected not to find content: \'button\' but continuously found it.`)
+
+          expect(containsLog.get('state')).to.eq('passed')
+          expect(containsLog.get('error')).to.be.undefined
+          expect(containsLog.get('$el').get(0)).to.eq(button.get(0))
+          const consoleProps = containsLog.invoke('consoleProps')
 
           expect(consoleProps.Yielded).to.eq(button.get(0))
           expect(consoleProps.Elements).to.eq(button.length)
 
+          expect(assertionLog.get('state')).to.eq('failed')
+          expect(err.message).to.include(assertionLog.get('error').message)
```

Comment:
Get .get('button') is no longer marked as failed because it resolved correctly and the assertion was the failure.
|Before | After| 
| -- | -- |
|<img width="488" alt="Screen Shot 2022-11-02 at 7 19 57 PM" src="https://user-images.githubusercontent.com/14099737/199626047-b4cacafa-4a68-4a28-9d88-9ed89244d769.png">| <img width="450" alt="Screen Shot 2022-11-02 at 7 15 18 PM" src="https://user-images.githubusercontent.com/14099737/199626002-bbc42c72-53aa-46df-af8e-c15c2cf631a6.png">|